### PR TITLE
Consistency Fixes

### PR DESCRIPTION
**Pull request by JuneGame**

### CODE REVIEW

File: ITEM.tsv [hunk 1]
```diff
@@ -1145,7 +1145,7 @@ ITEM_20150317_001144	Capria Gloves
 ITEM_20150317_001145	Follower Leather Armor
 ITEM_20150317_001146	Mummyghast Mail 
 ITEM_20150317_001147	{memo X}Vejo Boots 
-ITEM_20150317_001148	Mien Boots 
+ITEM_20150317_001148	$Mine Boots
```
Comment (ttgmichael) [inline ITEM.tsv:1148]: It's actually Mien Boots. Not a misspelling xD.

File: ITEM.tsv [hunk 2]
```diff
@@ -1767,29 +1767,29 @@ ITEM_20150317_001766	Empty_Boots
...
+ITEM_20150317_001774	$Large HP Potion 
+ITEM_20150317_001775	$Restores 640 HP. {nl}Restores HP every 2 seconds for 15 seconds. {nl}Right-click to use. {memo Cooldown: 60 seconds} 
+ITEM_20150317_001776	$Small SP Potion 
+ITEM_20150317_001777	$Restores 144 SP. {nl}Restores SP every 2 seconds for 15 seconds. {nl}Right-click to use. 
+ITEM_20150317_001778	$Restores 240 SP. {nl}Restores SP every 2 seconds for 15 seconds. {nl}Right-click to use. {memo Cooldown: 45 seconds} 
+ITEM_20150317_001779	$Large SP Potion 
+ITEM_20150317_001780	$Restores 480 SP. {nl}Restores SP every 2 seconds for 15 seconds. {nl}Right-click to use. {memo Cooldown: 60 seconds} 
+ITEM_20150317_001781	$Small Stamina Pill 
+ITEM_20150317_001782	$Restores 15 Stamina. {nl}Right-click to use. {memo Cooldown: 30 seconds} 
+ITEM_20150317_001783	$Stamina Pill
```
Comment (ttgmichael) [inline ITEM.tsv:1783]: Memo X lines are labeled by IMC to tell us that the lines are not being used.

Do not remove those tags. Heck, don't erase any memo lines, you don't see them in game and they're just there as additional side notes.

File: ITEM.tsv [hunk 3]
```diff
@@ -1767,29 +1767,29 @@ ITEM_20150317_001766	Empty_Boots
...
-ITEM_20150317_001787	Small Recovery Potion 
-ITEM_20150317_001788	Restores 244 HP and SP. {nl}Restores HP and SP every 2 seconds for 15 seconds. {nl}Right-click to use. Cooldown of 30 seconds. 
-ITEM_20150317_001789	Recovery Potion 
-ITEM_20150317_001790	Restores 373 HP and SP. {nl}Restores HP and SP every 2 seconds for 15 seconds. {nl}Right-click to use. Cooldown of 30 seconds. 
-ITEM_20150317_001791	Large Recovery Potion 
-ITEM_20150317_001792	Restores 744 HP and SP. {nl}Restores HP and SP every 2 seconds for 15 seconds. {nl}Right-click to use. Cooldown of 30 seconds. 
+ITEM_20150317_001770	$Small HP Potion 
+ITEM_20150317_001771	$Restores 192 HP. {nl}Restores HP every 2 seconds for 15 seconds. {nl}Right-click to use. 
+ITEM_20150317_001772	$HP Potion 
+ITEM_20150317_001773	$Restores 320 HP. {nl}Restores HP every 2 seconds for 15 seconds. {nl}Right-click to use. {memo Cooldown: 45 seconds}
```
Comment (ttgmichael) [inline ITEM.tsv:1773]: Why are you adding a memo tag for the Cooldown description? These tags are unseen in the game and thus you are technically removing the description from the game.

Just say

> Cooldown: 45 seconds.

**Comment by JuneGame** [comment]
My apologies. I'll try to fix everything soon according to standards.

Thank you for the guidelines.

**Pull request closed without merging**